interfaces/snap_app_set.go

```diff
@@ -196,35 +193,33 @@ func (a *SnapAppSet) SlotRunnables(slot *ConnectedSlot) []snap.Runnable {
 //   - "snap.$snap_instance.{$app1,...$appN, $hook1...$hookN}" if there are
 //     some, but not all, apps/hooks bound
 //   - "snap.$snap_instance.*" if all apps/hooks are bound to the plug or slot
-func labelExpr(apps []*snap.AppInfo, hooks []*snap.HookInfo, appSet *SnapAppSet) string {
+func labelExpr(appSet *SnapAppSet, connected []snap.Runnable) string {
```

Comment:
this maybe could take directly the connected plug or slot using an interface of 
interface {
  AppSet()  ...
  Runnables() ...
}

which means could drop needing slot/plugLabelExpression on SnapAppSet again ?

Reply:
Done in eb33c90dc8fca2bd1864ddb34ee60356f1b3194c, this does get rid of some unneeded indirection.